web/turkey/survey/static/survey/js/auditors/within_typing_delay.js

```diff
@@ -0,0 +1,39 @@
+var AuditorWithinTypingDelay = {
```

Comment:
There is a before_typing_delay.js which returns the time of the first typing event. This file checks if the time of that first typing event is within some typing_delay window in which we expect them not to type.